### PR TITLE
Weights-Biases

### PR DESCRIPTION
**Pull request by anna1904**
1. Init Weights-Biases, added to training
2. Sweep config
3. Model card init
4. Model registry

### CODE REVIEW

File: Labeling/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+1. Run Docker image with label studio inside
+```angular2html
+docker run -it -p 8080:8080 -v `pwd`/mydata:/label-studio/data heartexlabs/label-studio:latest
+```
+2. Open the `http://localhost:8080` and signup/login to the Label studio
+3. Load 50 samples of images
+4. Write labels for categories and label data
```
Comment (truskovskiyk) [inline Labeling/README.md:8]: this should not be in this PR

File: WBlogger/wandb/run-20230507_220413-cj1vzzjb/files/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,273 @@
+aiohttp==3.8.4
```
Comment (truskovskiyk) [inline WBlogger/wandb/run-20230507_220413-cj1vzzjb/files/requirements.txt:1]: hm, please remove this folder from git

File: WBlogger/wandb/run-20230507_220413-cj1vzzjb/files/wandb-metadata.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (truskovskiyk) [inline WBlogger/wandb/run-20230507_220413-cj1vzzjb/files/wandb-metadata.json:1]: remove

File: WBlogger/wandb/run-20230507_220413-cj1vzzjb/files/wandb-summary.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"constant": 0.9, "_timestamp": 1683497881.076041, "_runtime": 8027.960459947586, "_step": 31, "train/loss": 2.1081, "train/learning_rate": 0.0, "train/epoch": 0.0, "train/global_step": 10, "eval/loss": 2.2892749309539795, "eval/accuracy": 0.164, "eval/runtime": 787.567, "eval/samples_per_second": 1.27, "eval/steps_per_second": 1.27, "train/train_runtime": 8016.9078, "train/train_samples_per_second": 0.001, "train/train_steps_per_second": 0.001, "train/total_flos": 774975459041280.0, "train/train_loss": 2.279929208755493, "_wandb": {"runtime": 8027}}
```
Comment (truskovskiyk) [inline WBlogger/wandb/run-20230507_220413-cj1vzzjb/files/wandb-summary.json:1]: remove from git

File: classification/classification/model_card.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    training_data="./train.py",
+    citation="cite ViT: Vision Transfomer article ",
+    authors=["Anna Konovalenko"],
+    codebase="",
+    version="v1.0",
+)
+
+# Save ModelCard
+card.save("model_card.md")
+print(card)
```
Comment (truskovskiyk) [inline classification/classification/model_card.py:31]: wrap as a function please

File: classification/classification/model_card.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
+from huggingface_hub import ModelCard, ModelCardData
+import wandb
+from pathlib import Path
+
+# Training code goes here...
```
Comment (truskovskiyk) [inline classification/classification/model_card.py:5]: hm, should we remove this comment?

File: classification/classification/model_card.md [hunk 7]
```diff
@@ -0,0 +1,206 @@
+---
```
Comment (truskovskiyk) [inline classification/classification/model_card.md:1]: is this template or end result? should we remove 

lot of 
```
[More Information Needed]
```

File: classification/classification/model_card.py [hunk 8]
```diff
@@ -0,0 +1,31 @@
+from huggingface_hub import ModelCard, ModelCardData
```
Comment (truskovskiyk) [inline classification/classification/model_card.py:1]: this is great!

**Review by truskovskiyk — APPROVED** [review]
@anna1904 amazing work! let's fix your training metrics please

**Comment by truskovskiyk** [comment]
@anna1904 this one:
image
